src/behaviors/emp/emp.js

```diff
@@ -1,22 +1,22 @@
 import Behavior from '../behavior.js';
 
-class ClaporRave extends Behavior {
+class Emp extends Behavior {
```

Comment:
Please use meaningful names and try to refrain from confusing abbreviations or shortened versions of words. While you might understand what they mean, other developers may not. `Emphasis` would be easier to understand.